bok_choy/page_object.py

```diff
@@ -534,6 +534,18 @@ def axs_audit_rules_to_run(self):
         """
         return []
 
+    def axs_audit_rules_to_ignore(self):
+        """
+        List of rules to ignore for accessibility errors on the page.
+        See https://github.com/GoogleChrome/accessibility-developer-tools/tree/master/src/audits
+
+        E.g. return ['badAriaAttributeValue']
+        An empty list means to run rules as defined by axs_audit_rules_to_run.
+        Otherwise, if rules are listed here, they will be ignored even if
+        they are specified in axs_audit_rules_to_run.
+        """
+        return None
```

Comment:
Maybe the default return value should be [] here rather than None, to keep consistency with axs_audit_rules_to_run. 

It is potentially confusing is that "None" has different meanings. If I'm understanding correctly, the value of axs_audit_rules_to_ignore overrides whatever you put in axs_audit_rules_to_run, UNLESS you set axs_audit_rules_to_run to None, in which case it doesn't matter what you put axs_audit_rules_to_ignore because no rules are checked.